lib/scrapPaperReader.js

```diff
@@ -1,5 +1,80 @@
+const config = require('./config');
+const path = require('path');
+const { v4: uuidv4 } = require('uuid');
+const fileStore = require('../lib/file-store');
+const fs = require('fs').promises;
+
+const quagga = require('quagga').default;
+const imagemagick = require('imagemagick');
```

Comment:
From https://github.com/rsms/node-imagemagick:

> Note: This code has been unmaintained for a long time. Please consider using the gm module instead.

I'd very much like to heed their advice.